tests/conftest.py

```diff
@@ -2455,3 +2459,39 @@ def format_failure(port, failure):
 # HACK: We are using set_do_not_care_scapy but it will be deprecated.
 if not hasattr(Mask, "set_do_not_care_scapy"):
     Mask.set_do_not_care_scapy = Mask.set_do_not_care_packet
+
+
+def run_logrotate(duthost, stop_event):
+    logger.info("Start rotate_syslog on {}".format(duthost))
+    while not stop_event.is_set():
+        try:
+            # Run logrotate for rsyslog
+            duthost.shell("logrotate -f /etc/logrotate.conf", module_ignore_errors=True)
+        except subprocess.CalledProcessError as e:
+            logger.error("Error: {}".format(str(e)))
+        # Wait for 60 seconds before the next rotation
+        time.sleep(60)
```

Comment:
do we want 30 seconds or 60 seconds for each iteration?

Reply:
Thanks for review, updated the description, should be 60s.
based on the local test, the swss/sairedis.rec and swss/swss.rec log size increased very fast during the test, and seems "delaycompress" not working well, so I created a thread to rotate the log every 30s first, based on test, 60s should also OK, so I modify the sleep value to 60s.